sympy/polys/polyroots.py

```diff
@@ -861,6 +862,31 @@ def roots(f, *gens,
     >>> roots([1, 0, -1])
     {-1: 1, 1: 1}
 
+    ``roots`` will only return roots expressible in radicals.
+    If the given polynomial has some or all of its roots inexpressible in radicals,
+    the result of ``roots`` will be incomplete or empty respectively.
```

Comment:
These lines should be wrapped to a consistent length. Same in other places.

I don't know what editor you use but most code editors can do this for you automatically.